src/tests/tests/storage.js

```diff
@@ -191,7 +193,26 @@
     );
   });
 
+  QUnit.test("DNT does not return as an action if user has chosen not to", (assert) => {
```

Comment:
hmm. this test works in the suite, but does not work on its own because I mistakenly depended on the leftover state of a previous test.

`QUnit.test("DNT does not cascade", (assert) => {` runs `storage.setupDNT(DOMAIN);`

because that marks down DOMAIN as having a DNT policy, the next test disabling it does test it. I'll add that setupDNT to the disable test explicitly.